WordPressKitTests/BlogServiceRemote+ActiveFeaturesTests.swift

```diff
@@ -0,0 +1,49 @@
+import XCTest
+@testable import WordPressKit
+
+class BlogServiceRemote_ActiveFeaturesTests: RemoteTestCase, RESTTestable {
+
+    private let siteID = NSNumber(value: 1001)
+    private let syncBlogWithFeaturesFilename = "sites-site-active-features.json"
+    private let syncBlogWithEmptyFeaturesFilename = "sites-site-no-active-features.json"
+
+    private var syncBlogEndpoint: String {
+        "/sites/\(siteID)"
+    }
+
+    private lazy var remote: BlogServiceRemoteREST = {
+        .init(wordPressComRestApi: getRestApi(), siteID: siteID)
+    }()
+
+    // MARK: Tests
+
+    func testSyncBlogParsesActiveFeatures() async throws {
+        stubRemoteResponse(syncBlogEndpoint, filename: syncBlogWithFeaturesFilename, contentType: .ApplicationJSON)
+
+        let blog = try await withCheckedThrowingContinuation { continuation in
+            remote.syncBlog { remoteBlog in
+                continuation.resume(returning: remoteBlog)
+            } failure: { error in
+                continuation.resume(throwing: error!)
+            }
+        }
+
+        let features = try XCTUnwrap(blog?.planActiveFeatures)
+        XCTAssertEqual(features.count, 3)
```

Comment:
I'm actually surprised I don't see `XCTUnwrap` more often. I'm guilty of not using it either though 😅